### ISSUE TITLE
Missing Column on old PG Version

### ISSUE THREAD

**Issue by lrweck** [dbeaver/dbeaver#11479]
#### System information: 
- Windows 10 19042.804
- DBeaver version: 7.3.5.202102141704

#### Connection specification:
- PostgreSQL 8.3.23
- JDBC
- Do you use tunnels or proxies (SSH, SOCKS, etc)? Yes, SSH 

#### Describe the problem you're observing:

I am getting a error while invalidating connection that makes it impossible to login.

```
ERROR:  column t.typcategory does not exist at character 290
STATEMENT:  SELECT t.oid,t.*,c.relkind,format_type(nullif(t.typbasetype, 0), t.typtypmod) as base_type_name, d.description
        FROM pg_catalog.pg_type t
        LEFT OUTER JOIN pg_catalog.pg_class c ON c.oid=t.typrelid
        LEFT OUTER JOIN pg_catalog.pg_description d ON t.oid=d.objoid
        WHERE t.typname IS NOT null
        AND t.typcategory <> 'A' AND c.relkind is null or c.relkind = 'c'
```

#### Steps to reproduce, if exist:

Connect to a PostgreSQL database with version 8.23.23

**Comment by LonwoLonwo** [comment]
Hello @lrweck 
Please take a screenshot to clarify the version of the base and driver.

![2021-02-25 09_32_45-DBeaver Enterprise 7 3 0 - last_update](https://user-images.githubusercontent.com/45152336/109113278-0ed59e80-774d-11eb-9c68-098736534099.png)

**Comment by lrweck** [comment]
Hi @LonwoLonwo 

![image](https://user-images.githubusercontent.com/26086198/109168125-451b1a00-775d-11eb-860b-081f409f2e9a.png)

**Comment by LonwoLonwo** [comment]
Ok, sorry.

First check "Read all data types" in PostgreSQL tab.

![2021-02-25 17_53_06-DBeaver Enterprise 21 0 0 - _TEST_ Script-33](https://user-images.githubusercontent.com/45152336/109171116-7f52de80-7792-11eb-8d09-10558ba33729.png)

And then "Test connection".

(This is also workaround for column problem)

**Comment by lrweck** [comment]
![image](https://user-images.githubusercontent.com/26086198/109171738-cf18b200-7760-11eb-8912-7ac3d741621c.png)

**Comment by lrweck** [comment]
Enabling "Read all Datatypes" makes it possible to login/connect and use DBeaver again.

**Comment by LonwoLonwo** [comment]
We have changed the process of adding typcategory column to request for reading database data types.
Previously, we simply checked the version of the base, but obviously, this method is not always working.
So now we will find with a separate SELECT, whether typcategory column is in the system table we need.

Fixed in 21.0.1.
Will be available [here](https://dbeaver.io/) at 2021-03-22

**Comment by uslss** [comment]
verified
